docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -170,6 +170,12 @@
             <td>MemorySize</td>
             <td>Memory page size.</td>
         </tr>
+        <tr>
+            <td><h5>partial-update.ignore-delete</h5></td>
```

Comment:
The config option `partial-update.ignore-delete` doesn't include the update_before ignore meaning.